src/ArcValidation/Validate/Critical/ValidateCvBase.fs

```diff
@@ -39,6 +39,10 @@ module CvBase =
         let affiliation<'T when 'T :> CvContainer> (personCvContainer : 'T) =
             property "Affiliation" personCvContainer
 
+        /// Validates a person's ORCID.
+        let orcid<'T when 'T :> CvContainer> (personCvContainer : 'T) = 
+            property "<  ORCID>" personCvContainer
```

Comment:
why the spaces?

Reply:
I don't know...
That's the way the ARCGraphModel function parses it.

Reply:
weird flex but k :D